test/sqllogictest/map.slt

```diff
@@ -83,3 +84,24 @@ SELECT '{"a"=>"hello there!}'::map[text=>text]
 
 query error nested map types not yet supported
 SELECT '{a=>{b=>c}}'::map[text=>map[text=>text]]
+
+# Test map operators.
```

Comment:
sils vous plait, can I get tests for...
- _NULL_ keys both with _NULL_ present and absent from the map (idk how _NULL_ is handled as map elements,  so pardon my ignorance if that's not possible)
- looking for keys on empty map
- _NULL_ on an empty map
- the empty string as a variant on all of the same tests as _NULL_

Reply:
NULL keys and empty maps do not parse as maps - I believe this covers the first three bullet points.

I have added tests for the empty string!